server/routes/testutils/appSetup.ts

```diff
@@ -61,16 +63,25 @@ class MockUserService extends UserService {
   }
 }
 
-function appSetup(route: Router, production: boolean, session: Record<string, unknown>): Express {
+export const flashProvider = jest.fn()
+
+function appSetup(
```

Comment:
file copied from typescript template and altered as this one allows `session` to be passed in too